client/src/api/SwaggerApi.ts

```diff
@@ -482,7 +482,7 @@ type ApiConfig<SecurityDataType> = {
 };
 
 class HttpClient<SecurityDataType> {
-  public baseUrl: string = "//localhost/";
+  public baseUrl: string = "//localhost:8642/";
```

Comment:
Ist das gewollt, dass die Serverportänderung eingecheckt werden soll?

Reply:
Das macht an der Stelle keinen Unterschied, könnte da auch das Wort Bongotrommel eintragen. Die baseUrl wird in der index.ts im Api Package immer überschrieben. Um nicht noch mehr Leute zu verwirren, habe ich den Port jetzt trotzdem entfernt.